eth/main.cpp

```diff
@@ -350,8 +350,8 @@ int main(int argc, char** argv)
 	po::options_description generalOptions("General Options", c_lineWidth);
 	generalOptions.add_options()
 		("db-path,d", po::value<string>()->value_name("<path>"), ("Load database from path\n(default: " + getDataDir().string() + ").\n").c_str())
-#if ETH_EVMJIT
-		("vm", "<vm-kind>  Select VM; options are: interpreter, jit or smart (default: interpreter)")
+#if defined(ETH_EVMJIT) || defined(ETH_HERA)
```

Comment:
I prefer to write it without `defined`. It's still correct and shorter.

Reply:
Updated.